### PR TITLE
Release `0.8.1`

### PR DESCRIPTION
**Pull request by SutarPrasad**
### Added

- Ability to convert from an old style NetCDF object store to the new Zarr based store format - [PR #967](https://github.com/openghg/openghg/pull/967)
- Updated `parse_edgar` function to handle processing of v8.0 Edgar datasets. [PR #965](https://github.com/openghg/openghg/pull/965)
- Argument `time_resolved` is added as phase 1 change for `high_time_resolution`, also metadata is updated and added deprecation warning. - [PR #968](https://github.com/openghg/openghg/pull/968)
- Added explicit backwards compatability when searching previous object stores containing the `high_time_resolution` keyword rather than `time_resolved` - [PR #990](https://github.com/openghg/openghg/pull/990)
- Added ability to pass additional tags as optional metadata through `standardising_*` and `transform_flux_data functions`. [PR #981](https://github.com/openghg/openghg/pull/981)

### Fixed

- `source` can be passed to `transform_flux_data` with the EDGAR parser; `date` isn't stored with the transformed EDGAR data, since this is used for choosing what data to add, but doesn't describe all of the data in the object store. Fixed bug due to string split over two lines in logging message - [#PR 1010](https://github.com/openghg/openghg/pull/1010)
- Problem where the zarr store check raised an error for empty stores, preventing new zarr stores from being created - [#PR 993](https://github.com/openghg/openghg/pull/993)
- Retrieval of level 1 data from the ICOS Carbon Portal now no longer tries to retrieve a large number of CSV files - [#PR 868](https://github.com/openghg/openghg/pull/868)
- Added check for duplicate object store path being added under different store name, if detected raises `ValueError`. - [PR #742](https://github.com/openghg/openghg/pull/904)
- Added check to verify if `obs` and `footprint` have overlapping time coordinates when creating a `ModelScenario` object, if not then raise `ValueError` - [PR #954](https://github.com/openghg/openghg/pull/954)
- Added fix to make sure data could be returned within a date range when the data had been added non-sequentially to an object store - [PR #997](https://github.com/openghg/openghg/pull/997)
- Replace references to old `supplementary_data` repository with `openghg_defs` - [PR #999](https://github.com/openghg/openghg/pull/999)
- Added call to synonyms for species while standardising - [PR #984](https://github.com/openghg/openghg/pull/984)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,7 +5,28 @@ All notable changes to OpenGHG will be documented in this file.
...
+- Added ability to pass additional tags as optional metadata through `standardising_*` and `transform_flux_data functions`. [PR #981](https://github.com/openghg/openghg/pull/981)
+
+### Fixed
+
+- `source` can be passed to `transform_flux_data` with the EDGAR parser; `date` isn't stored with the transformed EDGAR data, since this is used for choosing what data to add, but doesn't describe all of the data in the object store. Fixed bug due to string split over two lines in logging message - [#PR 1010](https://github.com/openghg/openghg/pull/1010)
+- Problem where the zarr store check raised an error for empty stores, preventing new zarr stores from being created - [#PR 993](https://github.com/openghg/openghg/pull/993)
+- Retrieval of level 1 data from the ICOS Carbon Portal now no longer tries to retrieve a large number of CSV files - [#PR 868](https://github.com/openghg/openghg/pull/868)
+- Added check for duplicate object store path being added under different store name, if detected raises `ValueError`. - [PR #742](https://github.com/openghg/openghg/pull/904)
```
Comment (rt17603) [inline CHANGELOG.md:25]:
```suggestion
- Added check for duplicate object store path being added under different store name, if detected raises `ValueError`. - [PR #904](https://github.com/openghg/openghg/pull/904)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,7 +5,28 @@ All notable changes to OpenGHG will be documented in this file.
...
+- Added check to verify if `obs` and `footprint` have overlapping time coordinates when creating a `ModelScenario` object, if not then raise `ValueError` - [PR #954](https://github.com/openghg/openghg/pull/954)
+- Added fix to make sure data could be returned within a date range when the data had been added non-sequentially to an object store - [PR #997](https://github.com/openghg/openghg/pull/997)
+- Replace references to old `supplementary_data` repository with `openghg_defs` - [PR #999](https://github.com/openghg/openghg/pull/999)
+- Added call to synonyms for species while standardising - [PR #984](https://github.com/openghg/openghg/pull/984)
```
Comment (rt17603) [inline CHANGELOG.md:30]: Missing the following PR #872  update (might have been lost in a merge conflict):

Original text:
```
 - Bug fix for passing calibration_scale as optional parameter to the parser function. [PR #872](https://github.com/openghg/openghg/pull/872)
```

Though suggest update:
```
 - Bug fix to ensure calibration_scale is passed as an optional parameter to parse_* functions. [PR #872](https://github.com/openghg/openghg/pull/872)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,7 +5,28 @@ All notable changes to OpenGHG will be documented in this file.
...
+## [0.8.1] - 2024-05-17
+
+### Added
+
+- Ability to convert from an old style NetCDF object store to the new Zarr based store format - [PR #967](https://github.com/openghg/openghg/pull/967)
+- Updated `parse_edgar` function to handle processing of v8.0 Edgar datasets. [PR #965](https://github.com/openghg/openghg/pull/965)
+- Argument `time_resolved` is added as phase 1 change for `high_time_resolution`, also metadata is updated and added deprecation warning. - [PR #968](https://github.com/openghg/openghg/pull/968)
+- Added explicit backwards compatability when searching previous object stores containing the `high_time_resolution` keyword rather than `time_resolved` - [PR #990](https://github.com/openghg/openghg/pull/990)
+- Added ability to pass additional tags as optional metadata through `standardising_*` and `transform_flux_data functions`. [PR #981](https://github.com/openghg/openghg/pull/981)
+
```
Comment (rt17603) [inline CHANGELOG.md:19]: Missing the following update from PR #952 

```
- More explanation regarding use of `search_*` and `get_*` function in tutorial 1 [PR #952](https://github.com/openghg/openghg/pull/952)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -5,7 +5,28 @@ All notable changes to OpenGHG will be documented in this file.
...
+### Fixed
```
Comment (rt17603) [inline CHANGELOG.md:20]: Should add additional line for PR #962 
```
- Check added object stores listed in the configuration which are in the previous format with a warning raised for this [PR #962](https://github.com/openghg/openghg/pull/962)
```

**Review by rt17603 — CHANGES_REQUESTED** [review]
Just a bit of small tidy up in the CHANGELOG.md file, then this should be ready to go.

File: CHANGELOG.md (hunk 3 above)
Comment (SutarPrasad) [reply CHANGELOG.md:19]: This is in release 0.8.0

File: CHANGELOG.md (hunk 2 above)
Comment (SutarPrasad) [reply CHANGELOG.md:30]: this is in release 0.7.1

File: CHANGELOG.md (hunk 3 above)
Comment (rt17603) [reply CHANGELOG.md:20]: Hmm this PR being merged is listed in the commit history for this branch which is why I highlighted it - any ideas why that might be as that should already be on main?

@SutarPrasad
[Merge pull request](https://github.com/openghg/openghg/pull/1011/commits/e28eff1d670b4f9d4d1d3fba86f28dcfff1f4615) https://github.com/openghg/openghg/pull/952 [from openghg/Iss338-search-and-get_obs_surfac…](https://github.com/openghg/openghg/pull/1011/commits/e28eff1d670b4f9d4d1d3fba86f28dcfff1f4615)
[e28eff1](https://github.com/openghg/openghg/pull/1011/commits/e28eff1d670b4f9d4d1d3fba86f28dcfff1f4615)

Perhaps the commit history has become a bit circular meaning the merge is showing up a few times.

**Pull request merged**